src/features/movieList/Movie/index.js

```diff
@@ -25,7 +25,7 @@ const Movie = ({ movieTitle, movieRating, votesNumber, movieYear, moviePosterApi
         </MovieTitleBox>
 
         <MovieYear>
-          {movieYear.slice(0, 4)}
+          {/* {movieYear.slice(0, 4)} */}
```

Comment:
I'm not sure but I think we'll need it. You probably got an error the first time the page was loaded and you had to comment on it? I did it this way because we only need to display the year instead of the exact date. Let me know if this comment is needed.